web/blog/tenzir-node-v4.26/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Tenzir Node v4.26: ASL Integration"
+slug: tenzir-node-v4.26
+authors: [lava]
+date: 2025-01-21
+tags: [release, node]
+comments: true
+---
+
+Tenzir Node v4.26 continues to leverage our native Parquet and OCSF
+capabilities by adding a new operator to write data directly to
+Amazon Security Lake (ASL).
+
+![Tenzir Node v4.26](tenzir-node-v4.26.excalidraw.svg)
+
+[github-release]: https://github.com/tenzir/tenzir/releases/tag/v4.26.0
+
+<!-- truncate -->
+
+## Amazon Security Lake
+
+The new `to_asl` operator allows users to write data directly to the
+Amazon Security Lake:
+
+```tql
+let $s3_uri = "s3://aws-security-data-lake-eu-west-2-lake-abcdefghijklmnopqrstuvwxyz1234/ext/tenzir_network_activity/"
+ 
+load_kafka "ocsf_data"
+read_ndjson
+where @name == "ocsf.network_activity"
+to_asl $s3_uri,
+  region="eu-west-2",
+  accountId="123456789012"
+```
+
+For more information, take a look at the [Integrations](/next/integrations/amazon/security-lake)
+page in our documentation.
+
+## Output Changes
+
+With this version, we made some changes to the default output format
+of the Tenzir Node in order to make it easier for downstream tooling
+to work with the data provided by a Tenzir Node.
+
+:::warning Potentially Breaking Change
+
+This may break automations that parse the JSON output of a Tenzir Pipeline.
+:::
+
+### TQL Output
+
+By default, the node now outputs pipeline results as native TQL data
+rather than JSON.
+
+TQL is a superset of JSON, but it includes support for native IP, subnet,
+timestamp and duration types which were formerly rendered as strings.
+Additionally, TQL also supports and outputs trailing commas by default.
+
+To restore the previous behavior, append the [`write_json`](/next/tql2/operators/write_json)
+operator to your pipeline or set a default sink to restore it globally:
```

Comment:
To restore the previous JSON output format, append the [write_json](https://github.com/next/tql2/operators/write_json) operator to your pipeline or configure a default sink: